NitroxServer/ConsoleCommands/Abstract/Command.cs

```diff
@@ -65,7 +166,25 @@ public void SendMessageToPlayer(Optional<Player> player, string message)
             }
         }
 
-        public void Notify(Optional<Player> player, string message)
+        /// <summary>
+        ///     Send a message to an existing player, otherwise logs it in the console
+        /// </summary>
+        public void SendMessage(Optional<Player> player, string message)
+        {
+            if (player.HasValue)
+            {
+                player.Value.SendPacket(new ChatMessage(ChatMessage.SERVER_ID, message));
+            }
+            else
+            {
+                Log.Info(message);
+            }
+        }
+
+        /// <summary>
+        ///     Send a message to both console and an existing player
+        /// </summary>
+        public void SendMessageToBoth(Optional<Player> player, string message)
```

Comment:
"Both" is ambiguous without reading the summary.  when originally reading the code I was thinking this was a message sent to both players.  I'm not sure that we need to call out that this writes to the log (as we do that throughout).  It is likely fine to remove the else from the method above and allow all to be written to info.